src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -62,6 +87,49 @@ def __call__(
         return_dict: bool = True,
         **kwargs,
     ):
+        r"""
```

Comment:
Can we maybe give `enable_attention_slicing` also a docstring ? as well as the disable function?